### PR TITLE
Fix manual swagger schemas with use of drf_yasg

### PR DESCRIPTION
**Pull request by mjoyce91**
Use `drf_yasg` for manual schemas since that's what we now use for swagger

### CODE REVIEW

**Comment by mjoyce91** [comment]
bump for review - @elizabeth-jimenez @scott062 @SidelineCory24

File: talentmap_api/available_tandem/views/available_tandem.py [hunk 1]
```diff
@@ -40,13 +42,12 @@ class Meta:
...
-            coreapi.Field("page", location='query', type='integer', description='A page number within the paginated result set.'),
-            coreapi.Field("limit", location='query', type='integer', description='Number of results to return per page.'),
-            coreapi.Field("ordering", location='query', type='integer', description='Ordering'),
-        ]
-    )
+    @swagger_auto_schema(
+        manual_parameters=[
+            openapi.Parameter('page', openapi.IN_QUERY, type=openapi.TYPE_INTEGER, description='A page number within the paginated result set.'),
+            openapi.Parameter('limit', openapi.IN_QUERY, type=openapi.TYPE_INTEGER, description='Number of results to return per page.'),
+            openapi.Parameter('ordering', openapi.IN_QUERY, type=openapi.TYPE_STRING, description='Ordering')
```
Comment (elizabeth-jimenez) [inline talentmap_api/available_tandem/views/available_tandem.py:49]: noice

File: talentmap_api/fsbid/views/available_positions.py [hunk 2]
```diff
@@ -61,48 +61,48 @@ class FSBidAvailablePositionsTandemListView(BaseView):
...
+
+    @swagger_auto_schema(
+        manual_parameters=[
             # Pagination
-            coreapi.Field("ordering", location='query', description='Ordering'),
-            coreapi.Field("page", location='query', description='Page Index'),
-            coreapi.Field("limit", location='query', description='Page Limit'),
+            openapi.Parameter("ordering", openapi.IN_QUERY, type=openapi.TYPE_STRING, description='Ordering'),
+            openapi.Parameter("page", openapi.IN_QUERY, type=openapi.TYPE_STRING, description='Page Index'),
+            openapi.Parameter("limit", openapi.IN_QUERY, type=openapi.TYPE_STRING, description='Page Limit'),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/views/available_positions.py:70]: page and limit are `TYPE_INTEGER` elsewhere - should we keep it consistent?

**Review by elizabeth-jimenez — APPROVED** [review]
Just that small name change 👍

**Review by elizabeth-jimenez — APPROVED** [review]
👍

**Pull request merged**